graph/skills/index.py

```
@@ -0,0 +1,268 @@
+            conn.execute(
+                """
+                INSERT INTO skills_fts
+                    (name, description, prompt_template, tools_used, source_session_id, created_at)
+                VALUES (?, ?, ?, ?, ?, ?)
+                """,
+                (name, description, prompt_template, tools_str, source_session_id, created_at),
+            )
+            conn.commit()
+            log.debug("[skills] indexed skill: %s", name)
+        except sqlite3.Error as exc:
+            log.error("[skills] failed to index skill %s: %s", name, exc)
```

Ruff (0.15.10)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)